kubernetes/lacchain/webapp-service.yaml

```diff
@@ -6,8 +6,12 @@ metadata:
   name: monitor-webapp
 spec:
   ports:
```

Comment:
@kuronosec  it seems you are specifying `ports` twice

Reply:
ups, fixed @xavier506